plugins/ua_config_default.c

```diff
@@ -222,7 +222,21 @@ addEndpoint(UA_ServerConfig *conf,
     endpoint->transportProfileUri =
         UA_STRING_ALLOC("http://opcfoundation.org/UA-Profile/Transport/uatcp-uasc-uabinary");
     endpoint->securityMode = securityMode;
-    endpoint->securityLevel = (UA_Byte)securityMode;
+
+    /* A numeric value that indicates how secure the EndpointDescription is compared to other EndpointDescriptions
```

Comment:
Do you want to give a 1 for every policy with encryption?
Then it would be easier to check for != None.

Otherwise add a comment "in OPC UA v1.05 these are the policies considered safe".